Bundle shop token documentation (#15)
kkorotkov/NetLicensing-doc: docs/netlicensing-shop/index.md

```
{: .ml-5 }

> ## Shop Token Types

Reviewer: Not critical, but I'd suggest using a different term here than "Shop Token Type," since "Token Type" is clearly "SHOP" for all of these. For instance, it could be "Shop Token Kind."

> The NetLicensing Shop supports various token types, allowing vendors to implement specific license acquisition flows and improve the shopping experience for their customers.

Reviewer: ```suggestion
The NetLicensing Shop supports various kinds of tokens for opening a shop session, allowing vendors to implement specific license acquisition flows and improve the shopping experience for their customers.
```

### Licensee Shop Token

> This is the default token type, created for a specific licensee/customer. The identified customer can choose from the available modules and licenses. The aquired licenses will be assigned to the customer.

Reviewer: ```suggestion
This is the default token kind, created for a specific licensee/customer. The identified customer can choose from the available modules and licenses. The process considers the customer's existing licenses, and newly acquired licenses will be assigned to the customer.
```

<div>Request Parameters</div>
{: .code-example .ml-5 .code-header }
```http
tokenType=SHOP&licenseeNumber=%LICENSEE_NUMBER%
```
{: .ml-5 }

### Bundle Shop Token

> This token type can be created for a specific bundle, allowing customers to purchase only that bundle. The licenses acquired within the bundle will then be assigned to the customer.

Reviewer: ```suggestion
This token kind can be created for a specific bundle, allowing customers to purchase the entire bundle as a whole. The licenses acquired within the bundle will then be assigned to the customer.
```

<div>Request Parameters</div>
{: .code-example .ml-5 .code-header }
```http
tokenType=SHOP&licenseeNumber=%LICENSEE_NUMBER%&bundleNumber=%BUNDLE_NUMBER%&bundlePrice=%BUNDLE_PRICE%
```
{: .ml-5 }

### Permanent Shop Token

> This non-expiring token type allows the acquisition of predefined shopping cart items (such as a Pricing Table plan) identified by one or more “licenseTemplateNumber_N” values.

Reviewer: ```suggestion
This non-expiring token kind allows the acquisition of predefined shopping cart items (such as a Pricing Table plan) identified by one or more “licenseTemplateNumber_N” values. It is similar to a bundle, except that the specific license templates are defined directly in the token properties.
```
